src/main/antlr4/imports/mysql_partition.g4

```diff
@@ -0,0 +1,68 @@
+grammar mysql_partition;
+
+import mysql_idents;
+
+partition_by:
+  PARTITION BY
+  partition_by_what
+  partition_count?
+  subpartition_by?
+  partition_definitions?
```

Comment:
it.. might, if it was `skip_parens?`  generally I've tried to only use the skip-parens code when I'm *sure* there's parenthesis to be skipped (ie they're anchored to another expression).  The whole partition definition stuff here is optional, so yeah, could maybe be something else.